src/neuroglancer/segmentation_graph/source.ts

```diff
@@ -30,12 +32,35 @@ export enum VisibleSegmentEquivalencePolicy {
   NONREPRESENTATIVE_EXCLUDED = 1 << 2, // filter out non representative elements when iterating over visible segments
 }
 
+export class SegmentationGraphSourceTab extends Tab {
+  constructor(public layer: SegmentationUserLayer) {
+    super();
+    const {element} = this;
+
+    element.classList.add('neuroglancer-annotations-tab');
```

Comment:
this class adds display flex which is needed for my annotation list to render properly. Setting the property on the parent inside the graphene provided dependent view requires a !important to override existing display: contents.

Reply:
I took a look at this again and I think I was mistaken, it seems display flex needs to be on the neuroglancer-tab-content div.

I think tabContents needs to be able to add/toggle classes on the tab element. I could pass the tab or the element to tabContents.